docs/ReferenceManyFieldBase.md

```diff
@@ -0,0 +1,449 @@
+}: {
+    source: string;
+}) => {
+    const context = useListController();
+
+    if (context.isPending) {
+        return <p>Loading...</p>;
+    }
+
+    if (context.error) {
+        return <p className="error">{context.error.toString()}</p>;
+    }
+    return (
+        <p>
+            {listContext.data?.map((datum, index) => (
+                <li key={index}>{datum[source]}</li>
+            ))}
+        </p>
+    );
+};
+
+const AuthorShow = () => (
+    <Show>
+        <SimpleShowLayout>
+            <TextField source="first_name" />
+            <TextField source="last_name" />
+            <ReferenceManyFieldBase reference="books" target="author_id" >
+              <CustomAuthorView source="title" />
+            </ReferenceManyFieldBase>
+        </SimpleShowLayout>
+    </Show>
+);
+```
+
+`<ReferenceManyFieldBase>` accepts a `reference` attribute, which specifies the resource to fetch for the related record. It also accepts a `source` attribute which defines the field containing the value to look for in the `target` field of the referenced resource. By default, this is the `id` of the resource (`authors.id` in the previous example).
+
+You can also use `<ReferenceManyFieldBase>` in a list, e.g. to display the authors of the comments related to each post in a list by matching `post.id` to `comment.post_id`:
+
+```jsx
+import { List, DataTable, ChipField, ReferenceManyFieldBase, SingleFieldList } from 'react-admin';
+
+export const PostList = () => (
+    <List>
+        <DataTable>
+            <DataTable.Col source="id" />
+            <DataTable.Col source="title" />
+            <DataTable.Col label="Comments by">
+                <ReferenceManyFieldBase reference="comments" target="post_id">
+                    <CustomAuthorView source="name"/>
+                </ReferenceManyFieldBase>
+            </DataTable.Col>
+            <DataTable.Col>
+                <EditButton />
+            </DataTable.Col>
+        </DataTable>
+    </List>
+);
+```
+
+## Props
+
+| Prop           | Required | Type                                                                              | Default                          | Description                                                                         |
+| -------------- | -------- | --------------------------------------------------------------------------------- | -------------------------------- | ----------------------------------------------------------------------------------- |
+| `children`     | Required if no render | `Element`                                                                         | -                                | One or several elements that render a list of records based on a `ListContext`      |
+| `render`     | Required if no children | `(listContext) => Element`                                                                         | -                                | Function that receives a `ListContext` and render elements      |
```

Comment:
Same remark as in `ReferenceFieldBase`: replace the "required if no render|children" by "Optional" with an asterisk, and then, at the end of the list, add an explanation that either children or render is required.